.github/workflows/annotations.yaml

```diff
@@ -0,0 +1,58 @@
+name: annotations
+
+on:
+  push:
+    branches:
+      - future
+      - main
```

Comment:
Great, have you tested whether this works on your own fork? If you add it to your own fork and then do a PR on top of it, this should run too, so you should be able to test that it works properly :)